developer/js/lexical-model-compiler/lexical-model.ts

```diff
@@ -45,6 +52,44 @@ interface LexicalModelCompiledTrie extends LexicalModelCompiled {
   trie: string;
 }
 
+/**
```

Comment:
This seems to be duplicated from worker-interfaces.ts. We should avoid repetititition :)

Reply:
Yeah, I thought at one point we'd wired in some of the *.d.ts files from the LMLayer as references, reusing those definitions.

Then again, maybe it's that I wanted to do that originally but _didn't_, since that was when this code was in a separate repo.  We can definitely link to those .d.ts files now, though, and it makes sense for the compiler to be aware of worker-interfaces.ts's contents, as that's what models have to target.

Reply:
> Then again, maybe it's that I wanted to do that originally but _didn't_, since that was when this code was in a separate repo. We can definitely link to those .d.ts files now, though, and it makes sense for the compiler to be aware of worker-interfaces.ts's contents, as that's what models have to target.

What is the least bad way to do this in TypeScript? A simple `/// <ref> source` ?

Reply:
Yep.  If from /developer/js/index.ts:

`/// <reference path="../../common/predictive-text/worker/worker-interfaces.ts" />`

will compile in the definitions there.  KeymanWeb does something similar to link in the LMLayer.